### ISSUE TITLE
Incorrect error on nested list

### ISSUE THREAD

**Issue by jacqueslelezard** [IDPF/epubcheck#681]
On the toc.xhtml, into the nav list you should be able to use nested list like
```
<nav>
 <h2>Title</h2>
 <ol>
  <li>
   <ul>
     <li>
      <a href="sample.xhtml">
     </li>
   </ul>
 <li>
</ol>
```
But it seems you can't put anything else than `a` or `span` in `li` element. I am missing something or is it a bug?

**Comment by tofi86** [comment]
What's the error message?

I think you can only use `<ol>` but not `<ul>`, but besides that, nested lists are allowed afaik!

The error may also come from the first list entry which has no label but only a sublist.

**Comment by mattgarrish** [comment]
You can have a nested ordered list, but the list item that contains it still requires a label in an &lt;a> or &lt;span>:

``` html
<li><a href="s01.xhtml">Section 1</a>
    <ol>
        <li><a href="...">Section 1.1</a></li>
        ...
    </ol>
</li>
```

**Comment by jacqueslelezard** [comment]
The bug is the following :
```
ERROR (RSC-005) at "epub_ref.epub/OEBPS/toc.xhtml" (line 12, col 25):
   Error while parsing file 'element "ol" not allowed yet; expected element "a" or "span"'. 
```
Note :
-  I  updated my file to use `ol` instead of `ul`
- I'm using Pagina Epub Checker (graphical interface using epubcheck 4.0.1)

**Comment by jacqueslelezard** [comment]
Ok, I finally made it with your comments: apparently you MUST put a `a` or `span` before introducing a new list and you MUST NOT use `ul`, only `ol` .
Is it the right behaviour? (I don't understand why `ul` is forbidden)

**Comment by mattgarrish** [comment]
The ul element is for unordered lists. The table of contents reflects the ordered hierarchy of headings in the publication, so using an unordered list would be an odd choice.

The restrictions on structuring were put in to facilitate machine processing and rendering, as the table of contents is typically rendered in a specialized tree view by reading systems. That's the primary purpose of the navigation document.
